# This config file is used by ros2_control
controller_manager:
  ros__parameters:
    update_rate: 100  # Hz

    ruka_arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController


    ruka_hand_controller:
      type: joint_trajectory_controller/JointTrajectoryController


    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


    is_broadcaster:
      type: imu_sensor_broadcaster/IMUSensorBroadcaster
      
ruka_arm_controller:
  ros__parameters:
    joints:
      - base_link__link_01
      - link_01__link_02
      - link_02__link_03
      - link_03__link_04
      - link_04__link_05
      - link_05__link_06
    command_interfaces:
      - position
      - velocity
    state_interfaces:
      - position
      - velocity
ruka_hand_controller:
  ros__parameters:
    joints: 
      - link_hand_1__up_finger
    command_interfaces:
      - position
      - velocity
    state_interfaces:
      - position
      - velocity

is_broadcaster:
  ros__parameters:
    sensor_name: imu_sensor
    frame_id: link_06
